src/rust/id.shinigami/res/source.json

```
@@ -0,0 +1,18 @@
+{
+    "info": {
+        "id": "id.shinigami",
+        "lang": "id",
+        "name": "Shinigami",
+        "version": 1,
```

you're re-using the same id, so you need to update the version
```suggestion
        "version": 6,
```

---

done!